### DISCUSSION TITLE
Import-DbaCsv  - Path never found

### DISCUSSION THREAD

**Discussion by eliassal** [dataplat/dbatools#7934] [Q&A]
I execute the following statement
 Import-DbaCsv -SqlInstance SQL2K12 -Database DBA_Rep -AutoCreateTable -Path "SQLErrorLog.csv"
WARNING: [16:57:41][Import-DbaCsv] SQLErrorLog.csv cannot be found

I tried .\SQLErrorLog.csv as well as "C:\temp\SQLErrorLog.csv", no way alwyas getting

][Import-DbaCsv] C:\temp\SQLErrorLog.csv cannot be found

So the question, should I create an empty fiel before?
Thanks

**Comment by andreasjordan** [comment]
`Import-DbaCsv` is to import data from an existing file into the database. If you don't have a file - what do you want the command to do?

**Comment by eliassal** [comment]
Really Sorry, it was a misunderstanding on my side

**Comment by potatoqualitee** [comment]
@eliassal - it seems that perhaps you wanted to create a CSV file _from_ a database, to do that, you would probably want to use `Invoke-DbaQuery` and `Export-Csv`. Or if you want to export a whole database, explore `sqlpackage`.
